### ISSUE TITLE
remove or achive repo?

### ISSUE THREAD

**Issue by dimalinux** [noot/go-relayer#14]
Now that `atomic-swap` is no longer using `go-relayer`/`go-relayer-client`, I was wondering if the 2 repos should be removed from `AthanorLabs`?  We probably want to discourage anyone from using the them.  Possible options include deletion, making them private, moving them to a personal account, or changing the repo status to archived.

**Comment by noot** [comment]
why do we want to discourage anyone from using them? the code still works fine. I can move them to my personal account if you want?

**Comment by dimalinux** [comment]
> why do we want to discourage anyone from using them?

The code that we did use from this repo had to go through a lot of iterations until we got it right, and it is a tiny fraction of the code in the repo, so the rest hasn't been tested and vetted, which I don't see happening. Without a clear use case for current projects, there is no stimulus to put in the hard hours to bring the code to production and bear ongoing costs of maintenance.

In the end, we abandoned the gsnforwarder because it had no front-running protection. The only people to even ever use the mainnet version of the forwarder contract appears to be our old SwapFactory contract:
https://etherscan.io/address/0xB2b5841DBeF766d4b521221732F9B618fCf34A87
